### DISCUSSION TITLE
How do I build JammyPup using GitHub actions?

### DISCUSSION THREAD

**Discussion by RishonDev** [puppylinux-woof-CE/woof-CE#3774] [Q&A] [answered]
Hi! I have a repository fork on my account. when I go to the actions tab, and enable the jammy workflow, it doesn't show the workflow button. What am I doing wrong?

**Comment by dimkr** [comment]
https://github.com/puppylinux-woof-CE/woof-CE/wiki/Building-a-Puppy-on-GitHub

**Comment by lakshayrohila** [comment, marked as answer]
**Releases** is the workflow you are looking for. As @dimkr mentioned, please read https://github.com/puppylinux-woof-CE/woof-CE/wiki/Building-a-Puppy-on-GitHub. In case anything is unclear, you are always welcome to submit feedback/correct the wiki.
